int8 request
---                         # Demarcation
geometry_msgs/Point point    
